bedrock/firefox/templates/firefox/mobile/index.html

```diff
@@ -41,11 +46,13 @@ <h3>{{ _('Get automatic privacy on mobile') }}</h3>
           </div>
 
           <div class="mzp-c-hero-cta">
+          {% if variation == 'a' %}
```

Comment:
Unrelated to your question (which looks like pmac has already answered), but won't doing this hide the button for everyone not in the experiment?

You may also need to skip the functional tests for this page whilst the traffic cop experiment is running, because it alters the main CTA. I'd suggest pushing to `run-integration-tests` to make sure everything is ok.

Reply:
`run-integration-tests` appears to be okay now that the redirect tests are happy: https://gitlab.com/mozmeao/bedrock/pipelines/145756829

Reply:
-- I also updated this variation view to be !=b, thank you! :)

Reply:
> run-integration-tests appears to be okay now that the redirect tests are happy: https://gitlab.com/mozmeao/bedrock/pipelines/145756829

It entirely depends on the variation you get entered into. If I run the tests a few times locally, and get ?v=b then things fail. As such, you'll still need to skip the tests.

Reply:
@alexgibson oh I see. So this is blocked by  #8917 ?

Reply:
No, you just need to skip the tests here in this PR, no need to block.